Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -201,6 +201,14 @@ static wxString skip_efb_copy_to_ram_desc = wxTRANSLATE(
     "Stores EFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
     "in a small number of games.\n\nEnabled = EFB Copies to Texture\nDisabled = EFB Copies to RAM "
     "(and Texture)\n\nIf unsure, leave this checked.");
+static wxString skip_xfb_copy_to_ram_desc = wxTRANSLATE(
+  "Stores XFB Copies exclusively on the GPU, bypassing system memory. Causes graphical defects "
+  "in a small number of games that need to readback from memory.\n\nEnabled = XFB Copies to Texture\nDisabled = XFB Copies to RAM "
+  "(and Texture)\n\nIf unsure, leave this checked.");
+static wxString immediate_xfb_desc = wxTRANSLATE(
+  "Displays the XFB copy as soon as it is created.  Can cause graphical defects in some games if the game "
```

Comment:
When "immediate XFB" is checked, only a single XFB is displayed at a time.  That is one of the flaws with this approach, games that need to stitch together multiple XFBs (ex: the Rogue Squadron games) don't work with this feature.  So that is why I think we should keep it as "XFB copy".  Thoughts?